test/Microsoft.Azure.Devices.Edge.Test.Common/Windows/EdgeDaemon.cs

```diff
@@ -23,12 +17,23 @@ public EdgeDaemon(string scriptDir)
             this.scriptDir = scriptDir;
         }
 
-        public Task InstallAsync(
-            string deviceConnectionString,
-            Option<string> packagesPath,
-            Option<Uri> proxy,
-            CancellationToken token)
+        public async Task InstallAsync(string deviceConnectionString, Option<string> packagesPath, Option<Uri> proxy, CancellationToken token)
+        {
+            await this.InstallInternalAsync(deviceConnectionString, packagesPath, proxy, token);
+            await this.ConfigureAsync(proxy, token);
+        }
+
+        Task InstallInternalAsync(string deviceConnectionString, Option<string> packagesPath, Option<Uri> proxy, CancellationToken token)
```

Comment:
I like "InstallInternalAsync" convention; suggest to change "InternalStartAsync" and "InternalStopAsync" methods to "StartInternalAsync" and "StopInternalAsync" respectively.

Reply:
I *don't* like it, but I had to change the name because the signature is identical. If anything I'd rather use an underscore than the word "internal"--not sure why I didn't do it that way. But these are minor things, and I'm sure they'll change more than once as this work progresses. So I'd prefer to leave this alone for now.